packages/ui/components/toast/showToast.tsx

```diff
@@ -11,43 +11,51 @@ type IToast = {
 export const SuccessToast = ({ message, toastVisible }: IToast) => (
   <div
     className={classNames(
-      "data-testid-toast-success bg-brand-500 mb-2 flex h-9 items-center space-x-2 rounded-md p-3 text-sm font-semibold text-white shadow-md rtl:space-x-reverse",
+      "data-testid-toast-success bg-brand-500 mb-2 flex h-auto items-center space-x-2 rounded-md p-3 text-sm font-semibold text-white shadow-md rtl:space-x-reverse md:max-w-sm",
```

Comment:
I used `md:max-w-sm` because there is a case if the toast message is long(like in errors), the toast takes the full with of the viewport by default.